src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -33,6 +33,11 @@ public static void Aggregate_AggregateException_Wraps_OperationCanceledException
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(0, (i, j) => j, (i, j) => i, i => i));
             Functions.AssertAggregateAlternateCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate(() => 0, (i, j) => j, (i, j) => i, i => i));
+            Functions.AssertAggregateNotCanceled((token, canceler) => Cancel(token, canceler, source, operation).Aggregate((i, j) => j));
```

Comment:
@stephentoub - 
I'm going to think about it (and investigate) a bit more, but at the moment the answer will probably be "Unfortunately, for change purposes, yes" (otherwise I could theoretically ditch all these for just tests on the operator itself).

The reason has to do with `SequenceEqual` - if you add any operator before calling `SequenceEqual`, it nests any `AggregateExceptions` that occur (inside another one).  This includes when the exception is a not-cooperatively-canceled OCE.  No other operator has this behavior, and these tests document that.